gp-includes/misc.php

```diff
@@ -80,7 +80,7 @@ function gp_const_set( $name, $value ) {
 
 
 function gp_member_get( $object, $key, $default = '' ) {
-	return isset( $object->$key ) ? $object->$key : $default;
+	return is_string( $key ) && property_exists( $object, $key ) ? $object->$key : $default;
```

Comment:
We could wrap this in brackets too.

```suggestion
	return ( is_string( $key ) && property_exists( $object, $key ) ) ? $object->$key : $default;
```